### ISSUE TITLE
Maintained version

### ISSUE THREAD

**Issue by technyon** [a9183756-gh/Arduino-CMake-Toolchain#61]
Hi,

this projects seems to be not maintained now for over two years. Many issues are open and no pull requests are merged.

I've started to integrate a few pull requests and would continue to do so with other open issues / PRs:

https://github.com/technyon/Arduino-CMake-Toolchain

I was already able to compile with ESP32 core 2.0.3 which isn't possible with the code from the original repository.

Should one of the authors of the open PRs here read this, please reopen your PRs against my fork. Otherwise I'd see to manually integrate them.

**Comment by KSDaemon** [comment]
Hey @a9183756-gh if you have a minute — please come in and let's decide what to do: transfer ownership or give admin access to this repo at least to write a big banner that project development is now concentrated in another fork and archive this one. This will help other developers to find actual information and not create a bunch of forks. Thnx!

**Comment by matthew-james-brewer** [comment]
umm... I don't know what happened to @a9183756-gh. They haven't been active in 6 years.

**Comment by technyon** [comment]
I don't think so. I'm still open to maintain this, but there are better alternatives out there anyway I think.
